pages/common/md5sum.md

```diff
@@ -9,11 +9,11 @@
 
 - Calculate MD5 checksums for multiple files:
 
-`md5sum {{path/to/file1}} {{path/to/filen2}}`
+`md5sum {{path/to/file1}} {{path/to/file2}}`
```

Comment:
```suggestion
`md5sum {{path/to/file1 path/to/file2 ...}}`
```
I think this would apply here, as the description says it can calculate MD5 checksums for multiple files.

Reply:
Once, this is addressed I will update the Tamil translation here or in a new PR (once merged).

Reply:
I think we can get a small typo fix through and deal with that in a separate PR ^^